src/main/java/io/github/lottetreg/httpserver/HTTPInitialLine.java

```diff
@@ -0,0 +1,38 @@
+package io.github.lottetreg.httpserver;
+
+public class HTTPInitialLine {
+  public String method;
+  public String URI;
+  public String HTTPVersion;
+  private String whiteSpace = "\\s";
+
+  HTTPInitialLine(String initialLine) {
+      String[] splitInitialLine = initialLine.split(whiteSpace);
+
+      if(splitInitialLine.length != 3) {
+        throw new IncorrectlyFormattedInitialLineException(initialLine);
+      }
+
```

Comment:
What happens if a request is made with a different HTTP version like 1.1 for example?

Reply:
A different HTTP version won't make a difference... beside storing it, I'm pretty much ignoring the HTTP version and returning a response with a version of "1.0" by default